@types/intent-iq/index.d.ts

```diff
@@ -26,7 +26,8 @@ interface IntentIQConfig {
 	partnerClientId?: string;
 	partnerClientIdType?: number;
 	callback?: (data: IntentIqResponseData) => void;
-	browserBlackList?: string;
+	browserBlackList?: string[];
```

Comment:
```suggestion
        // TODO: remove type definition after new IIQ script is fully released
	browserBlackList?: string;
```
I guess this should remain a string?